src/component/Category/Category.js

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import { landingPageImages } from "../../assets/LandingPage/";
+import { Link } from "react-router-dom";
+import "./Category.css";
+
+export function Category() {
+  const {
+    collections: { jackets, travelbags, hikingbags, boots },
+  } = landingPageImages;
+  const {
+    brands: { bianyo, arnisa, celine, gucci },
+  } = landingPageImages;
+  const {
+    review: { john, lara, mohit },
+  } = landingPageImages;
+
+  return (
+    <div>
+      <section className="service">
+        <div className="box-container">
+          <div className="box">
+            <i className="fas fa-shipping-fast"></i>
+            <h3>Fast Delivery</h3>
+            <p>
+              Get your products delivered in superfast mode. Under 90 Mins at
+              unbelievable prices from the best sellers
+            </p>
+          </div>
+
+          <div className="box">
+            <i className="fas fa-undo"></i>
+            <h3>10 days replacements</h3>
+            <p>
+              Replacement is now easy with us. 10 days. Refund, Replacement or
+              Exchange.
+            </p>
+          </div>
+
+          <div className="box">
+            <i className="fas fa-headset"></i>
+            <h3>24 x 7 support</h3>
+            <p>
+              Available throughout the day, 6 days a week.This simply means that
+              the technical service is available around the clock.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section className="latest-products" id="products">
+        <h2 className="heading">Latest Collections</h2>
+
+        <div className="products-containerbox">
+          <div className="product-box">
+            <img src={jackets} alt="product" />
+            <div className="content-product">
+              <h3 className="desc-products">
+                Get your Jackets now at Best Price. For Limited Period Only.
+              </h3>
+              <Link to="./pages/products.html" className="btns">
+                Explore More
+              </Link>
+            </div>
+          </div>
+
+          <div className="product-box">
+            <img src={travelbags} alt="product" />
+            <div className="content-product">
+              <h3 className="desc-products">
+                Get your Travel Bags now at Best Price. Offer only for Today!
+              </h3>
+              <Link to="./pages/products.html" className="btns">
+                Explore More
+              </Link>
+            </div>
+          </div>
+
+          <div className="product-box">
+            <img src={hikingbags} alt="product" />
+            <div className="content-product">
+              <h3 className="desc-products">
+                Get your Hiking Bags now at Best Price. Grab it now.
+              </h3>
+              <Link to="./pages/products.html" className="btns">
+                Explore More
+              </Link>
+            </div>
+          </div>
```

Comment:
all this is duplicate code, you can use a `map()` and render the component, it will save a lot of lines of code.